.github/workflows/check-docstrings.yaml

```
@@ -0,0 +1,12 @@
+name: Check Docstrings
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  check-docstrings:
+    uses: catalystneuro/.github/.github/workflows/check_docstrings.yaml@docstring_tests
```

```suggestion
    uses: catalystneuro/.github/workflows/check_docstrings.yaml@main
```

I'd say in terms of file organization, if you could arrange things to work as above it would be preferred

---

I'd rather the workflows live in .github/workflows so that we can run them from the .github repo directly (very useful for development and debugging).